generators/app/index.js

```diff
@@ -161,6 +161,12 @@ module.exports = class extends Generator {
     this.installDependencies({
       bower: false
     });
+
+    // If commerce is chosen, install required dependency as well
+    if (this.props.services.indexOf('commerce') > -1) {
+      this.props.services.push('ct-types');
+    }
+
     this.npmInstall(this.props.services.map(service => `@cxcloud/${service}`));
```

Comment:
this would be nicer like this:

```js
const packages = this.props.services.map(service => `@cxcloud/${service}`);

if (this.props.services.includes('commerce')) {
	packages.push('@cxcloud/ct-types', '@buttercup/generator');
}

this.npmInstall(packages);
```